graphs/graph_modules/utils.py

```
@@ -0,0 +1,238 @@
+    
+    :param config: The configuration dictionary to validate
+    :return: A list of error messages, empty if no errors
+    """
+    errors = []
+    required_keys = ["TAUTULLI_API_KEY", "TAUTULLI_URL", "DISCORD_TOKEN", "CHANNEL_ID"]
+    for key in required_keys:
+        if key not in config or not config[key]:
+            errors.append(f"Missing or empty required configuration: {key}")
+    
+    if "UPDATE_DAYS" in config:
+        if not isinstance(config["UPDATE_DAYS"], int) or config["UPDATE_DAYS"] <= 0:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)